src/modules/rlm_rest/rlm_rest.c

```diff
@@ -135,6 +135,15 @@ static const CONF_PARSER xlat_config[] = {
 	CONF_PARSER_TERMINATOR
 };
 
+static const CONF_PARSER conn_config[] = {
+	{ FR_CONF_OFFSET("min", FR_TYPE_UINT32, rlm_rest_conn_config_t, min_elements), .dflt = "10" },
+	{ FR_CONF_OFFSET("max", FR_TYPE_UINT32, rlm_rest_conn_config_t, max_elements), .dflt = "100" },
+	{ FR_CONF_OFFSET("limit", FR_TYPE_BOOL, rlm_rest_conn_config_t, limit), .dflt = "no" },
+	{ FR_CONF_OFFSET("cleanup_interval", FR_TYPE_TIME_DELTA, rlm_rest_conn_config_t, interval), .dflt = "30s" },
+	{ FR_CONF_OFFSET("connect_timeout", FR_TYPE_TIME_DELTA, rlm_rest_conn_config_t, connect_timeout), .dflt = "3.0" },
+	CONF_PARSER_TERMINATOR
+};
+
```

Comment:
It may be better to move this into our libcurl library.  I think there's already some generic CONF_PARSER arrays in there.  I think the conn config is going to be pretty much identical for all the curl based modules?

Reply:
Makes sense - should be identical for IMAP and SMTP